.travis.yml

```diff
@@ -16,6 +16,7 @@ install:
 
 script:
   - make test
+  - git diff --exit-code
```

Comment:
Let add `git checkout go.mod go.sum` after `make init` above.

Reply:
Yeah. I was also thinking ignoring those files in diff. Your solution works better.